pkgs/build-support/docker/default.nix

```diff
@@ -845,6 +845,7 @@ rec {
         echo "         be better to only have one layer that contains a nix store."
 
         export NIX_REMOTE=local?root=$PWD
+        export USER=root
```

Comment:
Actually, I think it would be better to use "nobody" and add a comment to explain why it is required:/